src/providers/Onboarding.js

```diff
@@ -21,11 +21,7 @@ const DEFAULT_CONFIG = {
   agreement: {
     title: '',
     content: '',
-    challengePeriod: 0,
-    actionAmount: 0,
-    challengeAmount: 0,
```

Comment:
Ah nota that this fields are required.

The ones that can be derived are the stable amounts